examples/fixture/asm/r1cs/lc_return.asm

```diff
@@ -0,0 +1,3 @@
+@ noname.0.7.0
+
+v_1 + v_2 == (v_3) * (1)
```

Comment:
hopefully we'll see that with the reordering of public inputs it should be

```
v_1 + v_3 == v2
```

also what happened to write it like (A) * (B) = C ? it confuses me every time :D

Reply:
currently, it is C = (A) * (B)

it is intended to align with the order of assignment in the noname code :D
so when it is in debug mode, this asm can align with the statement like
c = a + b

but this alignment might not be that useful though.

Reply:
I guess it's a matter of taste :o doesn't really matter too much